FeatureTest.java

```diff
@@ -6,7 +6,7 @@ public static void main(String[] args) {
     public int square(int variable){
         String str = null;
         System.out.println(str.length());
-        str = "fj";
+        str = "fjff";
         return variable * variable;
     }
 }
```

Comment:
[code-reviewer[bot]]
Null pointer dereference: Variable 'str' is null (assigned null at line 7)